### ISSUE TITLE
Perf Experiment: --output-path is ignored

### ISSUE THREAD

**Issue by ebidel** [GoogleChrome/lighthouse#1170]
Running LH with  `--output-path=results.html --view` does not produce a report named `results.html`. The default naming is used:

<img width="547" alt="screen shot 2016-12-16 at 12 33 29 pm" src="https://cloud.githubusercontent.com/assets/238208/21277596/06bec876-c38c-11e6-9fb0-6bcece0b4be3.png">

**Comment by brendankenny** [comment]
`--output-path` is for the printed report. `--view` makes its own copy in its sub-directory. Is this confusing? If you were to open up your lighthouse directory you should only see `report.html`

**Comment by brendankenny** [comment]
I don't know what we're going to do for managing old `--view` reports, though

**Comment by ebidel** [comment]
Running:

    npm run fast -- --view --output-path=results.html http://localhost:8080/dbw_tester.html

I get 3 files :(

```
./lighthouse/localhost-8080_2016-12-16_13-53-36.report.html
./lighthouse/results.html
./lighthouse/lighthouse-cli/performance-experiment/src/reports/localhost-8080_2016-12-16_13-53-36.report.html
```

**Comment by brendankenny** [comment]
yeah, the `results.html` in that case is actually `--output=pretty`, I assume? That's one issue with a default of pretty output to stdout *and* html output to the working directory

**Comment by samuelli** [comment]
cc @WeiweiAtGit

**Comment by WeiweiAtGit** [comment]
Sorry for the confusing code.
`--view` creates another copy of report in `lighthouse-cli/performance-experiment/src/reports`. 
We did consider reusing the HTML file generated by `--output html` or `--output pretty`. But we chose to create a new copy because eventually we will have a different "report page" with some additional functionalities (e.g. add url block/cache patterns, compare different results). Also, reusing HTML file means we need to check the existence of the HTML file somewhere else in the code, which may not be obvious to other contributors when they make changes (e.g. if we decided to not produce html in pretty mode, we may forget to rewrite the HTML existence checking).
I probably should name it something like `--performance-experiment` instead of `--view` to reduce confusion.
